terraform-checksums.json

```diff
@@ -923,20 +923,6 @@
       "binary_checksum": "bac6bb1fb9383249b40afa46b83a4a78fa9f581347195f0e12bd1967d14a4823",
       "os": "linux",
       "arch": "arm64"
-    },
```

Comment:
Why are these being removed?

Reply:
So I can manually trigger and regenerate them. There has to be changes for the workflow to run and release.